<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtDemo::QtDemoMode</class>
 <widget class="QWidget" name="QtDemo::QtDemoMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>690</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">#Welcome--WelcomePage {
background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(247, 247, 247, 255), stop:1 rgba(215, 215, 215, 255));
}

QToolButton, QPushButton, QComboBox {
    border-image: url(:/welcome/images/btn_26.png) 4;
    border-width: 4;
    padding: 0px 6px;
    font-size: 12px;
}

*{
    color: black;
}

 QComboBox QAbstractItemView {
    background-color:white;
}

 QComboBox::down-arrow {
     image: url(:/welcome/images/combobox_arrow.png);
 }

QComboBox:drop-down
{
     subcontrol-origin: padding;
     subcontrol-position: top right;
     border-left-style: none; 
     border-top-right-radius: 1px;
     border-bottom-right-radius: 1px;
}

QToolButton:hover, QPushButton:hover, QComboBox:hover {
    border-image: url(:/welcome/images/btn_26_hover.png) 4;
}

QToolButton:disabled, QPushButton:disabled, QComboBox::disabled {
	color:gray;
}

QToolButton:pressed, QPushButton:pressed{
    border-image: url(:/welcome/images/btn_26_pressed.png) 4;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="gradientWidget" native="true">
     <property name="styleSheet">
      <string>#gradientWidget {
  background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(247, 247, 247, 255), stop:1 rgba(215, 215, 215, 255));
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="1" column="1">
       <widget class="QFrame" name="mainFrame">
        <property name="minimumSize">
         <size>
          <width>806</width>
          <height>606</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>814</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#mainFrame {
    border-image: url(:/welcome/images/background_center_frame.png) 4;
    border-width: 4;
}
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>18</number>
         </property>
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="widget">
            <layout class="QGridLayout" name="gridLayout_7">
             <property name="margin">
              <number>18</number>
             </property>
             <property name="spacing">
              <number>24</number>
             </property>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="bottomVerticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <spacer name="topVerticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3"/>
      </item>
      <item row="1" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_4"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
